app/controllers/home_controller.rb

```diff
@@ -11,6 +11,8 @@ def index
     # Feed artciles
     @articles = articles_for_current_page.page(params[:page]).per(6).padding(1)
 
+    @locale = LocaleService.find(locale: nil, lang_code: I18n.locale)
```

Comment:
If we put this here, it'll only work on the homepage.

We need to put into `ApplicationController` and in a `before_action`, so it gets called on every request.

Reply:
You're so damn right.. I was thinking about our discussion to only put it on the home page, but I put it in the header...